webapp/src/auth.ts

```diff
@@ -327,4 +371,19 @@ function clearState() {
     data.invalidate(NEEDS_SETUP);
 }
 
+function userPreferencesHandler(path: string): UserPreferences {
```

Comment:
>userPreferencesHandler [](start = 9, length = 22)

Can this be rolled into the auth virtual api? By the way, I want to rename this  to "user" api.

Reply:
Hmmm, we want to handle subfields like high-contrast and language. Can we nest multiple levels? Like "auth:user:preferences:high-contrast" ?